product/views/OrchestrationStack.yaml

```diff
@@ -88,3 +88,13 @@ graph:
 # Dimensions of graph (1 or 2)
 #   Note: specifying 2 for a single dimension graph may not return expected results
 dims: 
+
+col_formats:
+-
+-
+- :model_name
+-
+-
+-
+-
+-
```

Comment:
@chessbyte I believe the col_formats are positional where they align with the position of the corresponding column in `:cols`, `:col_order`, `:headings1, etc.